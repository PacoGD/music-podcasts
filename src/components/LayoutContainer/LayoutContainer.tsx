import { FunctionComponent, ReactNode } from "react";
import styles from "./style.module.css";

interface LayoutContainerProps {
  children: ReactNode;
}

export const LayoutContainer: FunctionComponent<LayoutContainerProps> = ({
  children,
}) => <div className={styles.layoutContainer}>{children}</div>;
