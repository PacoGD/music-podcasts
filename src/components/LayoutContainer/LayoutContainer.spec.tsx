import { render } from "@testing-library/react";
import { LayoutContainer } from "./LayoutContainer";

describe("LayoutContainer", () => {
  test("snapshot", () => {
    const { container } = render(
      <LayoutContainer>
        <div>Content</div>
      </LayoutContainer>
    );

    expect(container).toMatchSnapshot();
  });
});
