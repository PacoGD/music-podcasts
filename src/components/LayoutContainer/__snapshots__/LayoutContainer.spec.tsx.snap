// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutContainer snapshot 1`] = `
<div>
  <div>
    <div>
      Content
    </div>
  </div>
</div>
`;
